packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -188,6 +188,23 @@ class CloudFormation {
       });
   }
 
+  replaceTagVars(tagsArr) {
```

Comment:
Why is this here? Is it not possible to put this functionality where all the other tag functionality